src/plugins/intel_gpu/src/kernel_selector/tensor_type.cpp

```diff
@@ -885,7 +885,7 @@ NDims WeightsTensor::GetSimpleDims(const std::vector<size_t>& d, WeightsLayout l
 }
 
 WeightsTensor WeightsTensor::TransformIgnorePadding(WeightsLayout l, WeightsType t, size_t g, bool should_split) const {
-    const uint32_t src_channels = ChannelsCount(layout) - (DoesGroupDimExist(layout) ? 1 : 0);
+    const uint32_t src_channels = ChannelsCount(layout) - ((DoesGroupDimExist(layout))? 1 : 0);
```

Comment:
nit: why did you put additional parenthesis here? I think it is not necessary

Reply:
@isanghao Fixed it! Thanks!